functions/Export-DbaInstance.ps1

```diff
@@ -119,17 +130,24 @@ function Export-DbaInstance {
         PS C:\> Export-DbaInstance -SqlInstance sqlserver\instance
 
         All databases, logins, job objects and sp_configure options will be exported from
-        sqlserver\instance to an automatically generated folder name in Documents.
+        sqlserver\instance to an automatically generated folder name in Documents. For example, C:\Users\<username>\Documents\DbatoolsExport\<machinename>$sqlcluster-yyyyMMddHHmmss
```

Comment:
Please move this to one line. Word wrapping is handled by `Get-Help`.

For the example path, I would just put `%userprofile%\Documents...`. Please also make up a machine name and date value.